Makefile

```diff
@@ -132,6 +132,7 @@ COMMIT_COUNT := $(shell git rev-list --count HEAD)
 BASE_BUNDLE_VERSION ?= $(OPERATOR_VERSION)
 BUNDLE_VERSION ?= $(BASE_BUNDLE_VERSION)-$(COMMIT_COUNT)
 OPERATOR_IMAGE_REF ?= $(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)
+AIR_GAP_OPERATOR_IMAGE_REF ?= ""
```

Comment:
This var keeps image digest ref. Hence, how about to call it `OPERATOR_IMAGE_DIGEST_REF`? `AIR_GAP_OPERATOR_IMAGE_REF` is too high level abstraction here, it describes a use case. Image digest refs are used both in air gap and regular case. Its usage just enable image caching in a registry accessible behind firewall.

Reply:
Done. a4617d0de4c0f7b730b1054e06d9d384d8a4be11